swagger.yaml

```diff
@@ -0,0 +1,249 @@
+          schema:
+            $ref: "#/definitions/tweetsData"
+        204:
+          description: No content
+          schema:
+            $ref: "#/definitions/Error"
+        400:
+          description: Bad request
+          schema:
+            $ref: "#/definitions/Error"
+        404:
+          description: page not found
+          schema:
+            $ref: "#/definitions/Error"
+        405:
+          description: Invalid input
+          schema:
+            $ref: "#/definitions/Error"
+        default:
+          description: Unexpected error
+          schema:
+            $ref: "#/definitions/Error"
+
+    put:
+      tags:
+      - tweets
+      description: "update a tweet"
+      produces:
+      - application/json
+
+      parameters:
+        - name: tweetId
+          in: path
+          description: "update the tweet"
+          required: true
+          type: integer
+
+        - name: body
+          in: body
+          description: "tweet content to be updated"
+          required: true
+          schema:
+            $ref: "#/definitions/tweetsData"
+
+      responses:
+        200:
+          description: Successful response
+          schema:
+            $ref: "#/definitions/tweetsData"
+        204:
+          description: No content
+          schema:
+            $ref: "#/definitions/Error"
+        400:
+          description: Bad request
+          schema:
+            $ref: "#/definitions/Error"
+        404:
+          description: page not found
+          schema:
+            $ref: "#/definitions/Error"
+        405:
+          description: Invalid input
+          schema:
+            $ref: "#/definitions/Error"
+        default:
+          description: Unexpected error
+          schema:
+            $ref: "#/definitions/Error"
+
+    delete:
+      tags:
+        - tweets
+      description: "Delete a tweet"
+      produces:
+      - application/json
+
+      parameters:
+        - name: tweetId
+          in: path
+          description: "delete the tweet"
+          required: true
+          type: integer
+
+      responses:
+        200:
+          description: Successful response
+          schema:
+            $ref: "#/definitions/tweetsData"
+        204:
+          description: No content
+          schema:
+            $ref: "#/definitions/Error"
+        400:
+          description: Bad request
+          schema:
+            $ref: "#/definitions/Error"
+        404:
+          description: page not found
+          schema:
+            $ref: "#/definitions/Error"
+        405:
+          description: Invalid input
+          schema:
+            $ref: "#/definitions/Error"
+        default:
+          description: Unexpected error
+          schema:
+            $ref: "#/definitions/Error"
+
+
+definitions:
+  tweetsData:
+    properties:
+      userName:
+        type: string
+      tweetId:
+        type: integer
+      tweet:
+        type: string
+
+  Error:
```

Comment:
Same here, can you adhere to the JSON API error object schema?

* http://jsonapi.org/format/#errors
* https://github.com/osu-mist/web-api-skeleton/blob/master/swagger.yaml#L61-L79
* https://github.com/osu-mist/express-api-skeleton/blob/feature/CO-1192-express-api-skeleton/swagger.yaml#L170-L199